data-api/schemas/crm4.yml

```diff
@@ -152,16 +152,16 @@ Crm4QuoteDTO:
   type: object
   properties:
     companyName:
-      type: boolean
+      type: string
     expertName:
       type: string
     contactPhone:
       type: string
     costBasis:
       type: string
     preparationHours:
-      type: string
-      format: time
+      type: number
+      format: float
       description: only Time in 24 hrs format HH:MM
```

Comment:
I don't quite get it. If the type is a number, then it will not have the described format

Reply:
Done